OpenRA.Game/Settings.cs

```diff
@@ -23,24 +23,52 @@ public enum MouseScrollType { Disabled, Standard, Inverted }
 
 	public class ServerSettings
 	{
+		[Desc("Sets the server name.")]
 		public string Name = "OpenRA Game";
+
+		[Desc("Sets the internal port.")]
 		public int ListenPort = 1234;
+
+		[Desc("Sets the advertised port.")]
 		public int ExternalPort = 1234;
+
+		[Desc("Reports the game to the master server list.")]
 		public bool AdvertiseOnline = true;
+
+		[Desc("Locks the game with a password.")]
 		public string Password = "";
+
 		public string MasterServer = "http://master.openra.net/";
-		public bool DiscoverNatDevices = false; // Allow users to disable NAT discovery if problems occur
-		public bool AllowPortForward = true; // let the user disable it even if compatible devices are found
+
+		[Desc("Allow users to disable NAT discovery if problems occur.")]
+		public bool DiscoverNatDevices = false;
+
+		[Desc("Let the user disable UPnP even if compatible devices are found.")]
+		public bool AllowPortForward = true;
+
 		public bool NatDeviceAvailable = false; // internal check if discovery succeeded
+
 		public int NatDiscoveryTimeout = 1000; // ms to search for UPnP enabled NATs
-		public bool VerboseNatDiscovery = false; // print very detailed logs for debugging
+
+		[Desc("Print very detailed logs for debugging port forwarding in routers.")]
+		public bool VerboseNatDiscovery = false;
+
+		[Desc("Starts the game with a default map. Input as hash. Can be optained by the utility.")]
 		public string Map = null;
+
+		[Desc("Takes a comma separated list of IP addresses that are not allowed to join.")]
 		public string[] Ban = { };
 		public int TimeOut = 0;
+
+		[Desc("Run in headless mode.")]
 		public bool Dedicated = false;
+
+		[Desc("Automatically restart when a game ends.")]
```

Comment:
I've never understood why this is even a parameter... why would somebody ever want this to be false?